cmd/config/config.go

```diff
@@ -146,18 +150,14 @@ func getRawConfig() (map[string]interface{}, *viper.Viper, error) {
 func getViperWithConfigFile() (*viper.Viper, error) {
 	v := viper.GetViper()
 	if err := v.ReadInConfig(); err != nil {
-		if _, ok := err.(viper.ConfigFileNotFoundError); ok {
-			newViper := viper.New()
-			newViper.SetConfigFile(config.GetConfigFile())
-			err = newViper.WriteConfigAs(config.GetConfigFile())
-			if err != nil {
-				return nil, err
-			}
-
-			return newViper, nil
+		newViper := viper.New()
```

Comment:
It simplifies things to assume any reasonable error is because the file doesn't exist, so we'll try to write a new one regardless of the type of error returned.